packages/tradershub/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useAvailableMT5Accounts } from '@deriv/api';
+import { Provider } from '@deriv/library';
+import { Button, Heading, useBreakpoint } from '@deriv/quill-design';
+import { Modal } from '../../../../components/Modal';
+import { DummyComponent } from '../../components/DummyComponent';
+import { DynamicLeverageContext } from '../../components/DynamicLeverageContext';
+import { Jurisdiction, MarketType, MarketTypeDetails } from '../../constants';
+import { DynamicLeverageScreen, DynamicLeverageTitle } from '../../screens/DynamicLeverage';
+import { JurisdictionScreen } from '../../screens/Jurisdiction';
+
+const JurisdictionModal = () => {
+    const [selectedJurisdiction, setSelectedJurisdiction] = useState('');
+    const [isDynamicLeverageVisible, setIsDynamicLeverageVisible] = useState(false);
+    const [isCheckBoxChecked, setIsCheckBoxChecked] = useState(false);
+
+    const { show } = Provider.useModal();
+    const { getCFDState, setCfdState } = Provider.useCFDContext();
+
+    const { isLoading } = useAvailableMT5Accounts();
+    const { isMobile } = useBreakpoint();
+
+    const marketType = getCFDState('marketType') ?? MarketType.ALL;
+
+    const { title } = MarketTypeDetails[marketType];
+
+    const toggleDynamicLeverage = useCallback(() => {
+        setIsDynamicLeverageVisible(!isDynamicLeverageVisible);
+    }, [isDynamicLeverageVisible, setIsDynamicLeverageVisible]);
+
+    const jurisdictionTitle = `Choose a jurisdiction for your Deriv MT5 ${title} account`;
+
+    const JurisdictionFlow = () => {
+        if (selectedJurisdiction === Jurisdiction.SVG) {
+            return <DummyComponent />; // MT5PasswordModal
+        }
+
+        return <DummyComponent />; // Verification flow
+    };
+
+    useEffect(() => {
+        setCfdState('selectedJurisdiction', selectedJurisdiction);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [selectedJurisdiction]);
+
+    if (isLoading) return <Heading.H1>Loading...</Heading.H1>;
```

Comment:
Can you add a todo comment atleast just so we know where so we can upadate the component with the loading placeholder

Reply:
Done.